src/components/SubscriberHistory/SubscriberHistory.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { Helmet } from 'react-helmet';
+
+const SubscriberHistory = () => {
+  return (
+    <>
+      <FormattedMessage id="subscriber_history.page_title">
+        {(page_title) => (
+          <Helmet>
+            <title>{page_title}</title>
```

Comment:
Here I think we should add description as in the other pages.
```
<meta name="description" content={_('subscriberHistory_description')} />
```

Reply:
Maybe need to define the content because in production don't have it
![image](https://user-images.githubusercontent.com/8861133/69633985-0ebd8f00-1030-11ea-8c78-cc301e5b26a6.png)